src/main/java/org/painandsuffer/battle/Battle.java

```diff
@@ -0,0 +1,47 @@
+package org.painandsuffer.battle;
+
+import org.painandsuffer.battle.statuses.Status;
+import org.painandsuffer.characters.Creature;
+
+import java.util.Iterator;
+
+public class Battle {
```

Comment:
Consider creating an AI (class or method) that will use "random" or some kind of predefined strategies for better pvp simulation between Creatures. At the moment creatures are stupid orcs never mind their class just punching each other to the death... that's not what 'battle' between creatures was supposed to be, at least between sober ones.